src/convolution_api.cpp

```diff
@@ -90,6 +94,41 @@ extern "C" miopenStatus_t miopenSetConvolutionGroupCount(miopenConvolutionDescri
     return miopen::try_([&] { miopen::deref(convDesc).group_count = groupCount; });
 }
 
+extern "C" miopenStatus_t miopenSetConvolutionFindMode(miopenConvolutionDescriptor_t convDesc,
+                                                       miopenConvolutionFindMode_t findMode)
+{
+    MIOPEN_LOG_FUNCTION(convDesc, findMode);
+    return miopen::try_([&] {
+        miopen::deref(convDesc).findMode.Set(static_cast<miopen::FindMode::Values>(findMode));
+    });
+}
+
+extern "C" miopenStatus_t miopenGetConvolutionFindMode(const miopenConvolutionDescriptor_t convDesc,
+                                                       miopenConvolutionFindMode_t* findMode)
+{
+    MIOPEN_LOG_FUNCTION(convDesc, findMode);
+    return miopen::try_([&] {
+        miopen::deref(findMode) =
+            static_cast<miopenConvolutionFindMode_t>(miopen::deref(convDesc).findMode.Get());
+    });
+}
+
+// Hidden C++ functions for MIGraphX.
+namespace miopen {
+miopenStatus_t SetConvolutionFindMode(miopenConvolutionDescriptor_t convDesc, int findMode)
```

Comment:
This should be declared with extern `C`.

Reply:
@pfultz2 You need C functions?

Reply:
@pfultz2 
> But I should still be able to use `check_cxx_source_compiles` in cmake.

I thought you need C++, see https://github.com/ROCmSoftwarePlatform/MIOpen/pull/489#discussion_r502653632

Reply:
Done.